crates/gpui/src/app/entity_map.rs

```diff
@@ -576,7 +567,10 @@ pub struct WeakModel<T> {
 
 impl<T> std::fmt::Debug for WeakModel<T> {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-        f.debug_struct(type_name::<WeakModel<T>>()).finish()
+        f.debug_struct(&type_name::<Self>())
+            .field("entity_id", &self.any_model.entity_id)
+            .field("entity_type", &type_name::<T>())
+            .finish()
```

Comment:
FYI @mikayla-maki @maxbrunsfeld I'm assuming this conflicting impl just resulted from both `main` and this branch independently adding a `Debug` impl, and I resolved it by taking elements of both impls that seemed nice to me.